eng/tools/BaselineGenerator/Program.cs

```diff
@@ -311,7 +312,8 @@ private async Task<int> Run()
             {
                 var potentialLatestVersion = versionRange.FindBestMatch(
                     searchMetadata.Select(metadata => metadata.Identity.Version));
-                if (latestVersion == null || potentialLatestVersion.CompareTo(latestVersion) > 0)
+                if (latestVersion == null ||
+                    (potentialLatestVersion != null && potentialLatestVersion.CompareTo(latestVersion) > 0))
```

Comment:
`potentialLatestVersion` is `null` when the package isn't found for the current `PackageMetadataResource` i.e. in the referenced feed